setup.py

```diff
@@ -64,7 +64,7 @@
 
 install_requires = [
     'celery==3.1.17',
-    'Flask==0.10.1',
+    'Flask>=0.12',
```

Comment:
Also, now that we require Flask 0.12, the commit is no longer about docker only, so it would be better to change to the commit log message to something like `installation: upgrade to Flask 0.12`.